README.md

```diff
@@ -132,8 +132,7 @@ sudo chmod +x /usr/local/bin/cog
 Alternatively, you can build Cog from source and install it with these commands:
 
 ```console
-make
-sudo make install
```

Comment:
why this line removed?

Reply:
make   -- is the same as `make cog`
sudo make install -- just install cog binary to /usr/local/bin., it is static single file, no go involved.

Reply:
@dsingal0 Let's revert this part for now, since it's not related to the change you're proposing to use meaningful timestamps for temporary build directories. If you're getting unexpected errors when running the current commands, please open an issue and share the error you're seeing.